datafusion/sql/src/parser.rs

```diff
@@ -111,15 +116,28 @@ impl fmt::Display for CopyToStatement {
         let Self {
             source,
             target,
+            partitioned_by,
+            stored_as,
             options,
+            ..
         } = self;
 
         write!(f, "COPY {source} TO {target}")?;
+        if let Some(file_type) = stored_as {
+            write!(f, " STORED AS {}", file_type)?;
+        }
+        if !partitioned_by.is_empty() {
+            write!(f, " PARTITIONED BY ({})", partitioned_by.join(", "))?;
+        }
+
+        if self.has_header {
+            write!(f, " WITH HEADER ROW")?;
+        }
 
         if !options.is_empty() {
             let opts: Vec<_> = options.iter().map(|(k, v)| format!("{k} {v}")).collect();
             // print them in sorted order
```

Comment:
Is this comment accurate? I don't see an explicit sort here, is the `opts` vector somehow constructed in a sorted way?

Reply:
It was leftover.